utils/capellacontrol/controller.go

```diff
@@ -555,6 +555,53 @@ type CreateClusterRequest_Spec struct {
 	Services        []string                              `json:"services"`
 }
 
+type CreateOverRideClusterRequest struct {
```

Comment:
nitpick: `Override` is one word rather than two (`OverRide`).  Also, I think the path your using here (`/v2/organizations/%s/clusters/deploy`) implies that this method should be called something like `DeployCluster` (and thus the method would be `DeployClusterRequest`, where the fact that there is an override is a property _of_ the request, rather than being in the name of the request.

Reply:
yes, modified to DeployClusterRequest